i18n/en-US.properties

```diff
@@ -78,6 +78,16 @@ be.contentSidebar.activityFeed.comment.taskDeleteMenuItem = Delete task
 be.contentSidebar.activityFeed.comment.taskDeletePrompt = Are you sure you want to permanently delete this task?
 # Text to show on menu item to edit task
 be.contentSidebar.activityFeed.comment.taskEditMenuItem = Modify task
+# Title for x icon indicating someone rejected a task
```

Comment:
trying to remove this from the PR.  It got added after squashing for some reason.

Reply:
removed and pushed with `--no-verify`.  Think that worked.